flik/linesearch/test/test_conditions.py

```diff
@@ -1 +1,35 @@
 """Tests for flik.linesearch.conditions."""
+
+import numpy as np
+from nose.tools import assert_raises
+
+from conditions import check_armijo
+
+def test_armijo_pars():
+    """
+    Test for parameters in Armijo's condition
+    """
+    def fsq(var):
+        "Little function to square a vector"
+        return var**2
+    def grad(var):
+        "Gradient of the square function"
```

Comment:
it would be nice if you specify exactly the function that is being used. For example,
```python
"""Gradient of :math:`\sum_i x_i^2`"""
```